### ISSUE TITLE
Add Art Collection pages

### ISSUE THREAD

**Issue by jcbpl** [cityofaustin/austinconventioncenter.com#124]
The Convention Center needs to create an "Art Collection" page that shows a gallery of the artwork currently displayed in the building, along with the title and artist's name, which will link to an artist detail page.

#123 completed the necessary backend work to show captions alongside gallery thumbnails and to support rich text captions with links, so it's now possible to create the content for the Art Collection page using a gallery block:

<img width="750" alt="screenshot 2017-06-23 15 18 36" src="https://user-images.githubusercontent.com/3600/27498984-4a29e7aa-5827-11e7-992b-e4134993bb01.png">

If design direction requires custom styling of the thumbnail & caption grid, I'd recommend adding a conditional `acc-gallery-captions` class to the top-level div in [gallery.html][] `if block.showCaptions == true`. Then you can use that class in [gallery.scss][] to customize the appearance of galleries with inline captions.

The artist pages will include an artist photo and bio in a text block, and then include a gallery block to display the artist's work(s), with the "Show as full-width images" and "Always show captions" options enabled. I'd recommend marking up the photo and bio text using Markdown like this:

```
![Artist Name](artist-photo.jpg){:.acc-artist-photo}

Artist bio goes here...
```

And then using `.acc-artist-photo` to style the image size and margin. You may want to use the `$medium-screen` breakpoint to show the photo at full-width on small screens, and partial width on larger screens.

cc @benguhin. @mattlangan can also help to answer any questions about Contentful.

[gallery.html]: https://github.com/cityofaustin/austinconventioncenter.com/blob/master/_includes/components/content_blocks/gallery.html
[gallery.scss]: https://github.com/cityofaustin/austinconventioncenter.com/blob/master/_assets/stylesheets/components/gallery.scss

**Comment by sknep** [comment]
kthxbye :)

**Comment by mattlangan** [comment]
@sknep I think this has been merged in #123, yes? If so should we close this Issue?
